projects/angular/src/button/providers/button-group-focus.enum.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2016-2023 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export enum InitialFocus {
+  FIRST_ITEM = 'first', // Focus first item
+  LAST_ITEM = 'last', // Focus last item
```

Comment:
Nit: I don't think these comments are necessary. It's obvious what these enum values mean.